sys/kern/sysv_shm.c

```diff
@@ -352,6 +352,7 @@ kern_shmdt_locked(struct thread *td, const void * __capability shmaddr)
 {
 	struct proc *p = td->td_proc;
 	struct shmmap_state *shmmap_s;
+	static struct shmid_kernel *shmseg;
```

Comment:
Why do we need this to be static?

Reply:
Looks like a copy and past error and it needs to not be static
```suggestion
	struct shmid_kernel *shmseg;
```